extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/StaticResourcesRecorder.java

```diff
@@ -138,4 +138,15 @@ private boolean isCompressed(String path) {
         return contentType != null && compressMediaTypes.contains(contentType);
     }
 
+    public Handler<RoutingContext> additionalLocalDirectoryHandler(String root) {
+        StaticResourcesConfig config = httpConfiguration.getValue().staticResources;
+        return StaticHandler.create(root)
```

Comment:
We need to make sure there is no overlap or duplicates.

Also, having multiple static handler can lead to nasty classloading issues.